app/views/products/_product.html.erb

```diff
@@ -1,5 +1,10 @@
 <div class="col-sm-6 col-lg-4 pd-product-item">
   <div class="card">
+    <% if (product.status == "in stock")%>
+      <small class="text-success"><b><%= product.status%></b></small>
+    <% elsif (product.status == "out of stock")%>
```

Comment:
status gì mà "out of stock"

Reply:
trạng  thái  hết  hàng á anh

Reply:
em dùng string à sao không dùng enum

Reply:
Dạ không anh, em dùng  enum 
![image](https://user-images.githubusercontent.com/38943998/52956762-b089be80-33c2-11e9-9232-68eb058fc9bd.png)